src/Task.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Bref\Context\Context;
+use Bref\Event\Handler;
+
+class Task implements Handler
+{
+    public function handle(
+        $event,
+        Context $context,
+    ): array {
+        return $event;
```

Comment:
FIX: This example is going to get looked at every time someone new comes to this.

Make the demo task do something to the event rather than just return it.

I usually have a name property and return "Hello {$name}" but the specifics don't matter so much as making it a bit easier for the next guy to see the pattern.  Especially given the lack of type hint on $event.

Reply:
Agree, will do.